aws/rust-runtime/aws-sigv4/src/http_request/url_escape.rs

```diff
@@ -3,41 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
-use percent_encoding::{AsciiSet, CONTROLS};
-
-/// base set of characters that must be URL encoded
-const BASE_SET: &AsciiSet = &CONTROLS
-    .add(b' ')
-    // RFC-3986 §3.3 allows sub-delims (defined in section2.2) to be in the path component.
-    // This includes both colon ':' and comma ',' characters.
-    // Smithy protocol tests & AWS services percent encode these expected values. Signing
-    // will fail if these values are not percent encoded
-    .add(b':')
-    .add(b',')
-    .add(b'?')
-    .add(b'#')
-    .add(b'[')
-    .add(b']')
-    .add(b'@')
-    .add(b'!')
-    .add(b'$')
-    .add(b'&')
-    .add(b'\'')
-    .add(b'(')
-    .add(b')')
-    .add(b'*')
-    .add(b'+')
-    .add(b';')
-    .add(b'=')
-    .add(b'%');
-
-const QUERY_SET: &AsciiSet = &BASE_SET.add(b'/');
-const PATH_SET: &AsciiSet = BASE_SET;
+use aws_smithy_http::{label, query};
 
 pub(super) fn percent_encode_query(value: &str) -> String {
-    percent_encoding::percent_encode(value.as_bytes(), QUERY_SET).to_string()
+    query::fmt_string(value)
```

Comment:
Where were the `query::fmt_string` and `label::fmt_string` functions added?

Reply:
They were already defined in `aws-smith-http`. I'm using those ones.